cypress/integration/submitInvalidContactForm.spec.js

```diff
@@ -0,0 +1,74 @@
+describe('submitInvalidContactForm',function () {
+    const operations = [
+        {
+            // Estonian
+            contactPage: 'https://ut2019.voog.com/contact-1',
+            result: 'Vigane sisend',
+            language: "Estonian"
+
+        },
+        {
+            // English
+            contactPage: 'https://ut2019.voog.com/contact',
+            result: "Invalid input",
+            language: "English"
+        }
+    ];
+
+    const testing = [
+        //you can add your own tests here
+        {
+            email: ' ',
+            name: ' ',
+            message: ' '
+        },
+        {
+            email: 'qwe@qwe.com',
+            name: 'Tobias',
+            message: ' '
+        },
+        {
+            email: 'qwe@qwe.com',
+            name: ' ',
+            message: 'HALLOOOO'
+        },
+        {
+            email: ' ',
+            name: 'Tobias',
+            message: 'HALLOOOO'
+        },
+        {
+            email: ' ',
+            name: ' ',
+            message: 'HALLOOOO'
+        },
+        {
+            email: ' ',
+            name: 'Tobias',
+            message: ' '
+        },
+        {
+            email: 'qwe@qwe.com',
+            name: ' ',
+            message: ' '
+        }
+    ];
+
+    operations.forEach((operation) => {
+        it('Invalid '+operation.language+' form submission', function() {
+            cy.visit(operation.contactPage);
+
+            testing.forEach((tests) => {
+                cy.get('.contact-name input').type(tests.name);
+                cy.get('.contact-email input').type(tests.email);
+                cy.get('.contact-message textarea').type(tests.message);
+                cy.get('input[type="submit"]').click();
+                cy.get('.form_error').should('have.text', operation.result);
+                cy.get('.contact-name input').clear();
```

Comment:
We noticed that you clear the form's state after each test configuration execution, which is good, but really you should clear the form and prepare its clean state before the test. Otherwise if some test fails during the test and the clear actions are not performed then the next test would start with a dirty state. Maybe here it would be better to reload the page every time new test combination is being tested to ensure the state of the application is the cleanest.